### PR TITLE
chore: Change GitHub cli for CI

### PR DESCRIPTION
**Pull request by karasusan**

### CODE REVIEW

File: .yamato/upm-ci-publish-github-release.yml [hunk 1]
```diff
@@ -16,28 +16,42 @@ publish_com.unity.webrtc_github_release:
   name: Publish com.unity.webrtc to Github Release
   agent:
     type: Unity::VM
-    image: package-ci/win10:stable
+    image: package-ci/ubuntu:stable
     flavor: b1.large
+  variables:
+    GIT_TAG: Foo
```
Comment (kannan-xiao4) [inline .yamato/upm-ci-publish-github-release.yml:22]: Is this variables needed ?

Comment (kannan-xiao4) [reply .yamato/upm-ci-publish-github-release.yml:22]: I see, `GIT_TAG` value is `Foo`, is this correct?
I think `GIT_TAG` value need to set webrtc-version string (example `2.4.0-exp.1`).

Comment (karasusan) [reply .yamato/upm-ci-publish-github-release.yml:22]: Usually, we need to set the parameter on Yamato UI.
We don't have to change this line for every release.
image

Comment (karasusan) [reply .yamato/upm-ci-publish-github-release.yml:22]: OK, `Foo` may not suitable for us, so how about `You need to change this parameter on Yamato UI`?
Is it more clear than `Foo`?

Comment (karasusan) [reply .yamato/upm-ci-publish-github-release.yml:22]: > 
```suggestion
  variables:
    GIT_TAG: You need to change this parameter on Yamato UI
```

**Review by kannan-xiao4 — APPROVED** [review]

**Pull request merged**